lightning/src/routing/network_graph.rs

```diff
@@ -969,7 +1020,7 @@ impl NetworkGraph {
 	}
 
 	/// Marks a node in the graph as failed.
-	pub fn fail_node(&self, _node_id: &PublicKey, is_permanent: bool) {
+	pub fn fail_node(&self, _node_id: &NodeId, is_permanent: bool) {
```

Comment:
I think we should probably keep this as a `PublicKey`.

Reply:
Sorry, missed this comment. Will update in the morning.

Reply:
Ah, I see this was even unimplemented so totally my bad